graph/mongo/quadstore.go

```diff
@@ -128,25 +128,25 @@ func dialMongo(addr string, options graph.Options) (*mgo.Session, error) {
 	var dialInfo mgo.DialInfo
 	dialInfo.Addrs = strings.Split(addr, ",")
 	user, ok, err := options.StringKey("username")
-	switch {
-	case err != nil:
+	if err != nil {
 		return &mgo.Session{}, err
-	case ok:
+	}
+	if ok {
 		dialInfo.Username = user
 		password, ok, err := options.StringKey("password")
```

Comment:
Should it be inside "username present" if statement? User may specify just a password. Or it will not work for Mongo?